governance.md

```diff
@@ -0,0 +1,164 @@
+| Approver | Contributions acceptance approval| Highly experienced active reviewer and contributor to a subproject | [OWNERS] file approver entry|
+| Subproject owner | Set direction and priorities for a subproject | Demonstrated responsibility and excellent technical judgement for the subproject | [sigs.yaml] subproject [OWNERS] file *owners* entry |
+
+### Community groups
+The Op1st community is initiating Governance 1.0 with only one type of community group, Special Interest Groups (SIGs).
+The purpose of doing so is to empower SIGs to gather interested people around a subject area, then decide and implement that into governance.
+For example, one of the first acts of the Community SIG is to establish a milestone schedule for defining and holding elections for future steering committee(s).
+Governance 1.1 will therefore include details on new community groups (subprojects and working groups).
+
+#### SIGs
+Special Interest Groups (SIGs) are persistent open groups that focus on a part of the project. SIGs must have open and transparent proceedings.
+Anyone is welcome to participate and contribute provided they follow the Op1st Code of Conduct.
+The purpose of a SIG is to own and develop a set of subprojects.
+
+A SIG can have its own policy for contribution, described in a README or CONTRIBUTING file in the SIG folder in this repo (e.g. sig-community/CONTRIBUTING.md), and its own mailing list, chat channel, etc.
+
+#### Governance 1.1 future groups
+[These descriptions](https://github.com/kubernetes/community#governance) are borrowed from [Kubernetes governance around community groups](https://github.com/kubernetes/community/blob/master/governance.md#community-groups).
+They are good-enough placeholders for discussions.
+
+* **Committees** are named sets of people that are chartered to take on sensitive topics. This group is encouraged to be as open as possible while achieving its mission but, because of the nature of the topics discussed, private communications are allowed. Examples of committees include the Technical Steering Committee or an overarching Steering Committee, and things like security or code of conduct.
+* **Subprojects** Each SIG can have a set of subprojects. These are smaller groups that can work independently. Some subprojects will be part of the main Kubernetes deliverables while others will be more speculative and live in the kubernetes-sigs github org.
+* **Working Groups** are temporary groups that are formed to address issues that cross SIG boundaries. Working groups do not own any code or other long term artifacts. Working groups can report back and act through involved SIGs.
+* **User Groups** are groups for facilitating communication and discovery of information related to topics that have long term relevance to large groups of Kubernetes users. They do not have ownership of parts of the Kubernetes code base.
+See the full governance doc for more details on these groups.
```

Comment:
Is this level of subdivision actually warranted? Are you creating complexity before it is needed?

Reply:
Heh heh, especially when it's just a copy and paste still of the Kubernetes docs. I'll fix these in the next comment.

Fortunately, these are not being created or proposed at this point; this list is in a section labeled as `1.1` and `future`, and are placeholders for discussion. I'll add to the intro paragraph make it clear the discussion is intended to happen after Governance 1.0 is in use, and the Community SIG can begin the actual discussions.

Reply:
My fixes are in 48d2725:

> #### Governance 1.1 potential future groups
> [These descriptions](https://github.com/kubernetes/community#governance) are borrowed from [Kubernetes governance around community groups](https://github.com/kubernetes/community/blob/master/governance.md#community-groups).
> They are only placeholders for discussions, lightly edited.
> 
> **Subprojects** Each SIG can (or may? or should?) have a set of subprojects. These are smaller groups that can work independently. Some subprojects will be part of the main project Op1st deliverables while others will be more speculative.
> **Working Groups** are temporary groups that are formed to address issues that cross SIG boundaries. Working groups do not own any code or other long term artifacts. Working groups can report back and act through involved SIGs.
> **User Groups** are groups for facilitating communication and discovery of information related to topics that have long term relevance to large groups of Op1st users. They do not have ownership of parts of the Op1st code base, but they may contribute to other types of content and open works.

Reply:
Readable in context in my fork:  https://github.com/quaid/community/blob/main/governance.md#governance-11-potential-future-groups